app/controllers/miq_policy_controller/policy_profiles.rb

```diff
@@ -43,7 +43,7 @@ def profile_edit
         policies.each { |p| current.push(p.id) } # Build an array of the current policy ids
         mems = @edit[:new][:policies].invert     # Get the ids from the member list box
         begin
-          policies.each { |c| profile.remove_member(MiqPolicy.find(c)) unless mems.include?(c.id) } # Remove any policies no longer in the members list box
+          policies.each { |c| profile.remove_member(MiqPolicy.find_by(:id => c.id)) unless mems.include?(c.id) } # Remove any policies no longer in the members list box
```

Comment:
good catch :)

rubocop will probably warn this should be `find(c.id)` though

Reply:
Follow up to that .. is it the same as just `profile.remove_member(c)`? Or is `c` something other than MiqPolicy even though we're using its id as a policy id?

Reply:
@himdel `c` in this case will always be a MiqPolicy

Reply:
@himdel just to be safe i think it is better to have a find there. wdyt?

Reply:
@himdel please re-review, removed `.find` call as policy profiles can only have policies as their members